permutation/population_permutation.Rmd

```diff
@@ -80,10 +80,10 @@ Leave voters into another array, called `leave_ages`.
 # Extract Remain and Leave voters into their separate own data frames.
 # As above, we are just using Pandas here to get arrays for the
 # Remain and Leave voter ages.
-remainers = brexit_df[brexit_df['brexit_vote'] == 'Remain']
+remainders = brexit_df[brexit_df['brexit_vote'] == 'Remain']
```

Comment:
Ah - sorry - also `remainers` is correct here.

Reply:
yeap -- we saw it in parallel, fixed already, sorry for the sloppy original submission ;-)